Engine/Graphics/Outdoor.cpp

```diff
@@ -1250,10 +1250,19 @@ bool OutdoorLocation::Load(const String &filename, int days_played,
 
     pGameLoadingUI_ProgressBar->Progress();  //прогресс загрузки
 
-    assert(sizeof(Actor) == 836);
+    // assert(sizeof(Actor) == 836);
+    assert(sizeof(Actor) == 828 + 2 * sizeof(void *));
```

Comment:
arent we loading actors from file?

Reply:
Actors are being loaded from ODM / DDM. There's a pointer that is going through Games_LOD->LoadCompressed(map_filename). As per your request in next comment I will create Actor_MM7 and initialise Actor from there.

Reply:
Resolved.